ui/admin/app/components/ui/drawer.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+
+import * as React from "react";
+import { Drawer as DrawerPrimitive } from "vaul";
+
+import { cn } from "~/lib/utils";
+
+const Drawer = ({
```

Comment:
Pulled Combobox from shadcn example, came with a mobile responsive but can strip it out if necessary!